google-cloud-storage/src/main/java/com/google/cloud/storage/GrpcStorageImpl.java

```diff
@@ -502,7 +506,40 @@ public boolean delete(BlobId blob) {
 
   @Override
   public Blob compose(ComposeRequest composeRequest) {
```

Comment:
Can you also update the annotation in Storage.java to include `Transport.GRPC` in the supported transports for compose? https://github.com/googleapis/java-storage/blob/dc78526343c30ea85fd05c6977696309d687d958/google-cloud-storage/src/main/java/com/google/cloud/storage/Storage.java#L2499